st2client/st2client/commands/pack.py

```diff
@@ -190,12 +190,41 @@ def __init__(self, resource, *args, **kwargs):
                                  help='Force pack installation.')
 
     def run(self, args, **kwargs):
+        if args.packs:
+            if len(args.packs) == 1:
+                args.pack = args.packs[0]
+                pack_info = self.manager.search(args, **kwargs)
+                content = getattr(pack_info, 'content', {})
+                if content:
+                    print '\nFor "%s" pack following content will be registered:\n' % args.pack
+                    for entity in content.keys():
+                        if entity != 'tests':
+                            print '%s:  %s' % (entity, content[entity]['count'])
+                    print '\nIt may take a while based on number of items in content.'
+            else:
+                pack_content = {}
+                for pack in args.packs:
+                    args.pack = pack
+                    pack_info = self.manager.search(args, **kwargs)
+                    content = getattr(pack_info, 'content', {})
+                    for entity in content.keys():
+                        if entity != 'tests':
+                            if entity not in pack_content:
+                                pack_content[entity] = content[entity]['count']
```

Comment:
> And you can maybe just use a global list of available resources and print / include 0 in case pack contains no resource of that type.

Good idea! One of the resource is `tests`, we don't register that, right? I am not including that in the global list.